ProcessMaker/Models/Process.php

```diff
@@ -1216,4 +1218,61 @@ public function validateBpmnDefinition($addWarnings = false, &$warning = [])
         }
         return true;
     }
+
```

Comment:
Maybe you could move this section to ProcessTrait, so it could be added to Process and ProcessVersion

Reply:
I like the idea of the Trait, will try it out